pyproject.toml

```diff
@@ -68,21 +70,21 @@ module = [
     "bokeh.driving",
     "bokeh.embed.standalone",
     "bokeh.embed.wrappers",
-    "bokeh.ext",
-    "bokeh.io",
+    "bokeh.ext.*",
+    "bokeh.io", # TODO: .*;  mypy_path resolution is broken
```

Comment:
mypy is completely broken right now, when it comes to local `typings/` resolution. It's fixed, but we will need to wait for a release.

Reply:
mypy 0.971 is out, so hopefully this will be resolved soon.